docs/docsite/rst/user_guide/playbooks_debugger.rst

```diff
@@ -4,22 +4,26 @@
 Debugging tasks
 ***************
 
-Ansible offers a task debugger so you can try to fix errors during execution instead of fixing them in the playbook and then running it again. You have access to all of the features of the debugger in the context of the task. You can check or set the value of variables, update module arguments, and re-run the task with the new variables and arguments. The debugger lets you resolve the cause of the failure and continue with playbook execution.
+Ansible offers a task debugger so you can fix errors during execution instead of editing your playbook and running it again to see if your change worked. You have access to all of the features of the debugger in the context of the task. You can check or set the value of variables, update module arguments, and re-run the task with the new variables and arguments. The debugger lets you resolve the cause of the failure and continue with playbook execution.
 
 .. contents::
    :local:
 
-Invoking the debugger
+Enabling the debugger
 =====================
 
-There are multiple ways to invoke the debugger.
+The debugger is not enabled by default. If you want to invoke the debugger during playbook execution, you must enable it in one of three ways:
```

Comment:
I like the revision.  
I suggest a line break after the first sentence, making it the short introduction.  If you don't insert the line break, I recommend a period rather than a colon at the end of the second sentence.  Search engines often display the first couple of sentences in search results, and a complete sentence looks better than an incomplete stem.

Reply:
Thanks for the comment @bobjohnsrh - addressed in 43838b940bc74a05b8f745cb71f633dc3554db2f.